src/ChromeView.js

```diff
@@ -73,24 +75,34 @@ export default class ChromeView {
 
     this.showExtras();
 
-    this._$context.append(this._$view);
+    this._$view.removeClass(OFFSCREEN_CLASS);
   }
 
   renderSlide(slide) {
-    this._maybeHidePrevNext(slide);
+    this._maybeHidePrevNext();
+    this._appendSlide(slide);
 
-    const $next = $(`<div class="${CONTENT_CLASS} ${HIDDEN_CLASS}">`);
-    $next.html(this._getSlideContent(slide));
+    const $current = this._$contents.find('[data-slide-is-active]');
```

Comment:
Use js class for selection. Use data- for state of that selection. I'd use `.js-slide[data-is-active]`